### PR TITLE
feat(footer): Add tests

### PR DESCRIPTION
**Pull request by umahaea**
- Add test for footer
- Pointed to latest dependencies in pom.xml

@mhomolak @StommePoes @JasonLantz @wyseguyonline @udhadpa for your review.

### CODE REVIEW

File: src/test/java/compoundsTests/FooterTest.java [hunk 1]
```diff
@@ -0,0 +1,321 @@
...
+
+    @Test(testName = "Footer Styles Test", dataProvider = "Footer Styles Test Data", groups = {"desktop-ci", "desktop-regression"})
+    private void footerStylesTest(String size, int windowWidth, int windowHeight, By tocLinkElement, By copyRightTextElement, String[] expFontSize, String[] expLineHeight, By footerElement, String[] expMarginBottom, String setLight, String[] expColor, String device, ScreenOrientation mode) throws Exception {
+        String[] detailsPropertiesList = new String[]{"elementId", "footer-target", "componentName", "Footer"};
+        links = new LinkedHashMap<String, String>();
+        links.put("First link", "first");
+        links.put("Second link", "second");
+        links.put("Third link", "third");
+        linksArrayValue = buildListsArray(links);
+        String[] propsPropertiesList = new String[]{"copyrightText", "Pearson Education Inc", "light", setLight, "links", linksArrayValue};
```
Comment (StommePoes) [inline src/test/java/compoundsTests/FooterTest.java]: So does a color test run with light set to true and another for light set to false?

**Comment by StommePoes** [comment]
Seems okay. What's with the chain of replace with the JSON file? Isn't there a JSON creator/parser library for Java somewhere?

File: src/test/java/compoundsTests/FooterTest.java (hunk 1 above)
Comment (umahaea) [inline src/test/java/compoundsTests/FooterTest.java]: yes mallory, combination of true and false

**Comment by umahaea** [comment]
Regarding the chain of json replaces u see there, the original detail object is not a json object... I'm using the json object library to create a detail object and then use those replaceAll() to suit our original detail object..
Yes the code there looks lil funky and ugly, it is helping... in future If I can spend time to write it more cleaner I could..

**Comment by udhadpa** [comment]
Looks good.

**Comment by umahaea** [comment]
all approved. merging it.

**Pull request merged**